pkg/static/zz_generated_bindata.go

```diff
@@ -2,6 +2,7 @@
 // sources:
 // build/static/charts/traefik-10.3.001.tgz
 // build/static/charts/traefik-crd-10.3.001.tgz
+//go:build !no_stage
```

Comment:
Did go change their tags with the new version?

Reply:
yes, go 1.17 now automatically adds these build tags for you. I'm not a big fan but I assume they have their reasons.